_includes/privacy_policy_export.html

```diff
@@ -0,0 +1,202 @@
+</ol>
+</ol>
+<p><strong>Our use of Data</strong></p>
+<p><span style="font-weight: 400;">11. Any or all of the above Data may be required by us from time to time in order to provide you with the best possible service and experience when using our Website. Specifically, Data may be used by us for the following reasons:</span></p>
+<ol>
+<ol style="list-style-type: lower-alpha;">
+<li style="font-weight: 400;"><span style="font-weight: 400;">internal record keeping;</span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">improvement of our products / services;</span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">transmission by email of marketing materials that may be of interest to you;</span></li>
+</ol>
+</ol>
+<p><span style="font-weight: 400;">in each case, in accordance with this privacy policy.</span></p>
+<p><span style="font-weight: 400;">12. We may use your Data for the above purposes if we deem it necessary to do so for our legitimate interests. If you are not satisfied with this, you have the right to object in certain circumstances (see the section headed "Your rights" below).</span></p>
+<p><span style="font-weight: 400;">13. For the delivery of direct marketing to you via e-mail, we'll need your consent, whether via an opt-in or soft-opt-in:</span></p>
+<ol>
+<ol style="list-style-type: lower-alpha;">
+<li style="font-weight: 400;"><span style="font-weight: 400;">soft opt-in consent is a specific type of consent which applies when you have previously engaged with us (for example, you contact us to ask us for more details about a particular product/service, and we are marketing similar products/services). Under "soft opt-in" consent, we will take your consent as given unless you opt-out.</span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">for other types of e-marketing, we are required to obtain your explicit consent; that is, you need to take positive and affirmative action when consenting by, for example, checking a tick box that we'll provide.</span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">if you are not satisfied about our approach to marketing, you have the right to withdraw consent at any time. To find out how to withdraw your consent, see the section headed "Your rights" below.</span></li>
+</ol>
+</ol>
+<p><strong>Who we share Data with</strong></p>
+<p><span style="font-weight: 400;">14. We may share your Data with the following groups of people for the following reasons:</span></p>
+<ol>
+<ol style="list-style-type: lower-alpha;">
+<li style="font-weight: 400;"><span style="font-weight: 400;">our employees, agents and/or professional advisors - in order to facilitate the organisation and running of our events;</span></li>
```

Comment:
I'm not sure this covers volunteers - do we need to either update this or decide how we control the access of data?

Reply:
Adding a clause that includes volunteers